client/src/main/java/org/apache/celeborn/client/ShuffleClientImpl.java

```diff
@@ -1428,8 +1435,15 @@ private StatusCode getPushDataFailCause(String message) {
       cause = StatusCode.PUSH_DATA_CONNECTION_EXCEPTION_MASTER;
     } else if (message.startsWith(StatusCode.PUSH_DATA_CONNECTION_EXCEPTION_SLAVE.getMessage())) {
       cause = StatusCode.PUSH_DATA_CONNECTION_EXCEPTION_SLAVE;
+    } else if (message.startsWith(StatusCode.PUSH_DATA_TIMEOUT_MASTER.getMessage())) {
+      // Should behind PUSH_DATA_TIMEOUT
+      cause = StatusCode.PUSH_DATA_TIMEOUT_MASTER;
+    } else if (message.startsWith(StatusCode.PUSH_DATA_TIMEOUT_SLAVE.getMessage())) {
+      // Should behind PUSH_DATA_TIMEOUT
+      cause = StatusCode.PUSH_DATA_TIMEOUT_SLAVE;
     } else if (message.startsWith(StatusCode.PUSH_DATA_TIMEOUT.getMessage())) {
-      cause = StatusCode.PUSH_DATA_TIMEOUT;
+      // Convert PUSH_DATA_TIMEOUT to PUSH_DATA_TIMEOUT_MASTER in client side.
```

Comment:
We can delete PUSH_DATA_TIMEOUT, it's a little confusing to convert to Master/Slave in different sides.

Reply:
DOne